src/roi/Roi.ts

```diff
@@ -91,4 +138,452 @@ export class Roi {
   public getBorderPoints(options?: GetBorderPointsOptions): Array<Point> {
     return getBorderPoints(this, options);
   }
+
+  _computeBorderIDs(): { ids: number[]; lengths: number[] } {
+    const borders = getBorders(this);
+    this.#computed.borderIDs = borders.ids;
+    this.#computed.borderLengths = borders.lengths;
+    return borders;
```

Comment:
The ids and length should be in one computed property which returns an array of `{id: number, length: number}`

Todo in another PR